app/org/sagebionetworks/bridge/models/schedules/SchedulerFactory.java

```diff
@@ -0,0 +1,18 @@
+package org.sagebionetworks.bridge.models.schedules;
+
+import org.sagebionetworks.bridge.validators.ScheduleValidator;
+import org.sagebionetworks.bridge.validators.Validate;
+
+public class SchedulerFactory {
+
+    public static TaskScheduler getScheduler(String schedulePlanGuid, Schedule schedule) {
+        // TODO: This validation will be moved to create/update, not the time of use.
+        Validate.entityThrowingException(new ScheduleValidator(), schedule);
```

Comment:
Can we make a singleton ScheduleValidator (preferably a public static final in ScheduleValidator) instead of creating a new one every time?